apph-back/src/main/java/com/viseo/apph/dao/PhotoDao.java

```diff
@@ -23,4 +23,8 @@ public List<Photo> getUserPhotos(long idUser) throws NoResultException {
                 .setParameter("idUser", idUser)
                 .getResultList();
     }
+
+    public Photo getPhoto(long id) throws NoResultException {
+        return em.find(Photo.class, id);
+    }
```

Comment:
```suggestion
    public Photo getPhoto(long id) {
        return em.find(Photo.class, id);
    }
```

`em.find()` ne renvoie jamais de NoResultException, si il ne trouve rien il envoi un null.